examples/Notebooks/dev/flopy3_dev_mf6_packaging_distributing_flopy_plugins.ipynb

```diff
@@ -0,0 +1,234 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b4c70565",
+   "metadata": {},
+   "source": [
+    "# Distributing your FloPy Plugin as a Python Package\n",
+    "\n",
+    "FloPy plug-ins can be installed as python packages.  FloPy will automatically detect plug-ins in other python packages by using python package metadata.\n",
```

Comment:
```suggestion
    "FloPy plugins can be installed as python packages.  FloPy will automatically detect plugins in other python packages by using python package metadata.\n",
```

I don't think we want to hyphenate plugin.

Reply:
Fixed this everywhere.